lib/rubocop/cop/rspec/match_array.rb

```diff
@@ -17,14 +21,44 @@ module RSpec
       #
       #   # good
       #   it { is_expected.to match_array(%w(tremble in fear foolish mortals)) }
+      #
+      #   # bad
+      #   it { is_expected.to match_array([]) }
+      #   it { is_expected.to match_array(%w[]) }
```

Comment:
I had overlooked this 🙇‍♂️ I think this is the last one.
```suggestion
      #   it { is_expected.to match_array(%w()) }
      #   it { is_expected.to match_array(%i()) }
```

Reply:
There is also `%W` and `%I`. I meant to give only a few examples and not an exhaustive list :-)

As for `%w[]` vs `%w()`, [RuboCop defaults to enforcing](https://github.com/rubocop/rubocop/blob/856d73c467ddd07dcaa74a0f88aa94ae5ba4172a/config/default.yml#L4685-L4700) `[]`.

I hope it’s ok if I keep the PR as-is.